src/advanced/node-management.md

```diff
@@ -774,3 +779,4 @@ equal than `from` and less or equal than `to`.
 
 [closure]: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler
 [github_explorer]: https://github.com/exonum/exonum/blob/master/exonum/src/api/public/blockchain_explorer.rs
+[blockchain-state]: https://exonum.com/doc/glossary/#blockchain-state
```

Comment:
FYI: It may make sense to link to Markdown pages (i.e., `../glossary.md#blockchain-state` in this case); this is more flexible and makes links work in the Markdown viewers (e.g., the GitHub one).